src/features/retreat/components/auctioneer/AuctioneerModal.tsx

```diff
@@ -65,6 +66,11 @@ export const AuctioneerModal: React.FC<Props> = ({
     }
   }, [isOpen]);
 
+  const hasAuctionAccess = hasReputation({
+    game: gameState,
+    reputation: Reputation.Seedling,
```

Comment:
```suggestion
    reputation: Reputation.Grower,
```

Auction access is listed at Grower tier in reputation list